odoolint/tests/test_convert_file.py

```diff
@@ -178,6 +181,7 @@ def test_70_ref_unreachable_csv(self):
         self.create_unreachable('base.group_user')
         self.create_imd('security/ir.model.access.csv', 'data', 2, 1)
 
+    @unittest.skipIf(ODOO_PATCHED is True, "Odoo is patched with str2bool")
```

Comment:
@moylop260 After searching `odoolint_isolated` in addons-path I find it in `odoolintsqa/odoolint_isolated` I will make the change in that module but how I can prove it in runbot?

Regards.